### ISSUE TITLE
webpack error reported by mini-css-extract-plugin with AngularCompilerPlugin for aot

### ISSUE THREAD

**Issue by huineng** [swimlane/ngx-charts#1309]
**Describe the bug**
after installing charts (dependency of ngx-graph) and NOT using angular aot compiler ngToolsWebpack.AngularCompilerPlugin the project compiles.

using the ngToolsWebpack.AngularCompilerPlugin i get an error from mini-css-extract-plugin

```
ERROR in Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
TypeError: Cannot read property 'replace' of undefined
```
therefore i cannot compile ngx-charts to aot 

angular version 8.x
"@swimlane/ngx-charts": "^12.1.0",
 "@swimlane/ngx-graph": "^6.2.0",
 "webpack": "^4.41.2",

**Expected behavior**
compiles to aot

tried several combinations of loaders.. none worked

maybe this should be reported in ngx=graph to check how they include dependencies on ngx-charts
on the other side, if i want to use charts for angular aot, i will get the same error

**Additional context**
https://github.com/webpack-contrib/mini-css-extract-plugin/issues/186

**Comment by KingDarBoja** [comment]
@huineng I did managed to config some private company-related project which makes uses of `ngx-charts` with AOT compilation using AngularCompilerPlugin.

I will try to share it as soon as possible.

Cheers!

**Comment by KingDarBoja** [comment]
@huineng Here are my webpack config files, `common`, `dev` and `prod` in a [single gist](https://gist.github.com/KingDarBoja/591ee28eb1a04ee5bcac7f6906918a0d)

**Comment by marjan-georgiev** [comment]
Closing this. The library is compatible with AoT. We also released v13.0.0 recently, which is also compatible with Ivy.
